dashboard/db/migrate/20180703165502_change_blocks_to_utf8mb4.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeBlocksToUtf8mb4 < ActiveRecord::Migration[5.0]
+  def up
+    execute 'alter table blocks convert to charset utf8mb4 collate utf8mb4_unicode_ci'
+    execute 'alter table blocks modify name varchar(255) charset utf8mb4 collate utf8mb4_unicode_ci'
+    execute 'alter table blocks modify category varchar(255) charset utf8mb4 collate utf8mb4_unicode_ci'
+    execute 'alter table blocks modify config text(16383) charset utf8mb4 collate utf8mb4_unicode_ci'
+    execute 'alter table blocks modify helper_code text(16383) charset utf8mb4 collate utf8mb4_unicode_ci'
+  end
+
+  def down
+    execute 'alter table blocks convert to charset utf8 collate utf8_unicode_ci'
+    execute 'alter table blocks modify name varchar(255) charset utf8 collate utf8_unicode_ci'
+    execute 'alter table blocks modify category varchar(255) charset utf8 collate utf8_unicode_ci'
+    execute 'alter table blocks modify config text(21845) charset utf8 collate utf8_unicode_ci'
+    execute 'alter table blocks modify helper_code text(21845) charset utf8 collate utf8_unicode_ci'
```

Comment:
Rails annotates the model as `text(65355)` but MySQL wants the value to be the actual number of characters stored.  65355 / 3 = 21845.